### DISCUSSION TITLE
Several customizations

### DISCUSSION THREAD

**Discussion by Mecanik** [razonyang/hugo-theme-bootstrap#534] [Q&A] [answered]
Hello, I have the following questions and I thought it's best to put them into one discussion:

- How do you customize the loading bar colour on the page? Currently it's some multi-colour.
- The "blog" is called "posts", so you navigate to /posts/ and you see the posts. Can this be renamed to /blog/ somehow?
- How can you show archives on the side bar underneath tags?
- What is the Series intention? Sounds to me like a video section?
- How do you customise the 404 Not Found page? I would have shown 404 inside the actual layout - right now it's an empty page.
- I have my "own" comment system and this includes a .js file, on the previous project (testing) I was able to include .js/.css files on specific pages using Hugo rule "if", is there a way to do this here? 

Many thanks for all the help.

**Comment by razonyang** [comment, marked as answer]
## Loading bar color

Create a custom SCSS file and saved it as `assets/main/scss/_custom.scss`:

```scss
.loading-bar {
    background: red;
}
```

## Blog Section

Append any sections to `mainSections` on `config/_default/params.toml`:

```toml
mainSections = ["posts", "docs", "blog"]
```

So that the `blog` articles will be shown.

> You can create a blog article by `hugo new blog/new-post/index.md`.

## Archive on the sidebar

Currently, there is no archive widget on the sidebar, you can open a feature request about this, I'm not sure how this widget looks like.

## Series intention

I'm not sure. For me, it is a column for aggregating articles. Such as **K8s Guide**.

## 404 Page

This theme provides 404 pages, you can access it by `/404.html` or `/{lang}/404.html`.

> `{lang}` - the language key, such as `en`, `zh`...

## Comment Engine

See also [Custom Comments Widget](https://hbs.razonyang.com/en/docs/widgets/comments/#custom-comments-widget)
